From 6118225996b2baa5c98983ba5a9a5a1c22c2ac86 Mon Sep 17 00:00:00 2001
From: Huang Chenyi <b44402@freescale.com>
Date: Tue, 2 Jul 2013 10:39:51 +0800
Subject: [PATCH 210/304] ENGR00268651 Cannot boot up if connect Ethernet
 after data encryption

Connect Ethernet will start dhcp service
which class is late_start. Before umount tmpfs
on /data,it will only reset class main service.
The fact that dhcp service still exists causes
/data fail to umount. Change dhcp from late_start
to main will resolve this issue.

Signed-off-by: Huang Chenyi <b44402@freescale.com>
---
 imx6/etc/init.rc     | 6 +++---
 sabreauto_6q/init.rc | 2 +-
 sabresd_6dq/init.rc  | 2 +-
 3 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/imx6/etc/init.rc b/imx6/etc/init.rc
index 44d983a..5515b75 100755
--- a/imx6/etc/init.rc
+++ b/imx6/etc/init.rc
@@ -615,17 +615,17 @@ service mdnsd /system/bin/mdnsd
     oneshot
 
 service dhcpcd_wlan0 /system/bin/dhcpcd -ABKL
-    class late_start
+    class main
     disabled
     oneshot
 
 service dhcpcd_p2p /system/bin/dhcpcd -ABKL
-    class late_start
+    class main
     disabled
     oneshot
 
 service dhcpcd_eth0 /system/bin/dhcpcd -ABKL
-    class late_start
+    class main
     disabled
     oneshot
 
diff --git a/sabreauto_6q/init.rc b/sabreauto_6q/init.rc
index da26f1b..76a52d4 100644
--- a/sabreauto_6q/init.rc
+++ b/sabreauto_6q/init.rc
@@ -125,7 +125,7 @@ service ing /system/bin/ingsvcd -c /system/etc/gps/Orion.ini
     oneshot
 
 service iprenew_wlan0 /system/bin/dhcpcd -n
-    class late_start
+    class main
     disabled
     oneshot
 
diff --git a/sabresd_6dq/init.rc b/sabresd_6dq/init.rc
index 5f105d1..194b797 100755
--- a/sabresd_6dq/init.rc
+++ b/sabresd_6dq/init.rc
@@ -129,7 +129,7 @@ service ing /system/bin/ingsvcd -c /system/etc/gps/Orion.ini
     oneshot
 
 service iprenew_wlan0 /system/bin/dhcpcd -n
-    class late_start
+    class main
     disabled
     oneshot
 
-- 
1.8.0

